### PR TITLE
Updated to beta9 compatible

### PR DESCRIPTION
**Pull request by mstahv**
Probably done wrong, but at least I tried.

### CODE REVIEW

File: src/main/java/org/vaadin/pekka/WysiwygE.java [hunk 1]
```diff
@@ -61,7 +56,7 @@
 @HtmlImport("bower_components/wysiwyg-e/tools/justify.html")
 @HtmlImport("bower_components/wysiwyg-e/tools/heading.html")
 @HtmlImport("bower_components/wysiwyg-e/tools/blockquote.html")
-public class WysiwygE extends Component implements HasSize, HasStyle, HasValue<WysiwygE, String> {
+public class WysiwygE extends AbstractField<WysiwygE,String> implements HasSize, HasStyle {
```
Comment (pleku) [inline src/main/java/org/vaadin/pekka/WysiwygE.java]: I was planning on extending `AbstractSinglePropertyField` instead. Any reason why not choosing that one ?

File: src/main/java/org/vaadin/pekka/WysiwygE.java [hunk 2]
```diff
@@ -135,25 +131,25 @@ protected void initToolbar() {
...
-        getElement().setProperty(getClientValuePropertyName(), value);
+    protected void setPresentationValue(String newPresentationValue) {
+        getElement().setProperty(getClientValuePropertyName(), newPresentationValue);
     }
 
-    @Synchronize("blur")
-    @Override
-    public String getValue() {
+    @Synchronize(value = "blur", property = "value")
+    public String getText() {
```
Comment (pleku) [inline src/main/java/org/vaadin/pekka/WysiwygE.java]: I'd prefer the name of the value method to stay as _value_.

File: src/main/java/org/vaadin/pekka/WysiwygE.java (hunk 1 above)
Comment (mstahv) [reply src/main/java/org/vaadin/pekka/WysiwygE.java]: I had no idea of such a class due to the missing documentation ;-)

File: src/main/java/org/vaadin/pekka/WysiwygE.java (hunk 2 above)
Comment (mstahv) [reply src/main/java/org/vaadin/pekka/WysiwygE.java]: getValue is still there, but as it is implemented bit differently, the synchronize annotation don't seem to work anymore.

**Comment by pleku** [comment]
Closing PR as I migrated the component to be based on `AbstractSinglePropertyField` instead.

**Pull request closed without merging**